sycl/source/detail/persistent_device_code_cache.cpp

```diff
@@ -320,7 +320,7 @@ std::vector<std::vector<char>> PersistentDeviceCodeCache::getItemFromDisc(
 std::vector<std::vector<char>>
 PersistentDeviceCodeCache::getCompiledKernelFromDisc(
     const std::vector<device> &Devices, const std::string &BuildOptionsString,
-    const std::string SourceStr) {
+    const std::string &SourceStr) {
```

Comment:
nit: I think more idiomatic fix would be to use `std::string_view`.

Reply:
I am eliminating extra copy here, not intended to change types.